runelite-plugin.properties

```
@@ -1,6 +1,7 @@
 displayName=Goal Tracker
-author=Toofifty
-support=https://github.com/Toofifty/rl-goal-tracker
-description=Keep track of your goals and complete them automatically
-tags=task,to-do,todo
+author=toofifty
+support=https://github.com/toofifty/goaltracker
```

These should not be changed

```suggestion
author=Toofifty
support=https://github.com/Toofifty/rl-goal-tracker
```